### ISSUE TITLE
the difference between exuberant-ctags (OldC) parser and new C parser of u-ctags

### ISSUE THREAD

**Issue by masatake** [universal-ctags/ctags#971]
I'm testing new C parser by given the source code of linux kernel as input.
My script detects there is gap between the outputs.

```
[yamato@x201]/tmp% cat /tmp/baz.c 
typedef const struct pinfo {
	u8 opcode;
} INFO;
[yamato@x201]/tmp% u-ctags -o - --kinds-'*'='*' --languages=+OldC --language-force=OldC /tmp/baz.c
INFO	/tmp/baz.c	/^} INFO;$/;"	t	typeref:struct:pinfo	file:
opcode	/tmp/baz.c	/^	u8 opcode;$/;"	m	struct:pinfo	file:
pinfo	/tmp/baz.c	/^typedef const struct pinfo {$/;"	s	file:
[yamato@x201]/tmp% u-ctags -o - --kinds-'*'='*' --languages=+C --language-force=C /tmp/baz.c
INFO	/tmp/baz.c	/^} INFO;$/;"	t	typeref:typename:const struct pinfo{ u8 opcode; } 	file:
[yamato@x201]/tmp% 
```

**Comment by pragmaware** [comment]
Right. "const" confuses it. Will fix it tonight.

**Comment by pragmaware** [comment]
Fixed via #974
